### PR TITLE
Moved Instance Sizing to installation and moved Kafka Switch to Administration

### PR DESCRIPTION
**Pull request by hobbes-the-tiger**
these were fixed per feedback from @LH-ExcellentAdventure and @watsonian.

@LH-ExcellentAdventure moved the following:
/appendix/instance-sizing.md to /operations-guide/installation/instance-sizing.md
/operations-guide/configuration/basic-configuration/kafka-switch.md to /operations-guide/administration

Please make sure the redirects work as well the text that was changed. Thanks!

@mwl If you have time, please let me know I did things right, or if something needs to be fixed. thanks!

### CODE REVIEW

File: content/operations-guide/administration/_index.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ This content was created with a focus on clustered Humio environments.
 - [Disabling Ingest]({{< ref "disabling-ingest.md" >}})
 - [Updating your license]({{< ref "license-management.md" >}})
 - [Metrics]({{< ref "metrics.md">}})
+- [Switching Kafka servers]({{< ref "kafka-switch" >}})
```
Comment (mwl) [inline content/operations-guide/administration/_index.md]: When using [`ref`](https://gohugo.io/functions/ref/) it should be pointed towards a filename. So add `.md`, ie.
```
[Switching Kafka servers]({{< ref "kafka-switch.md" >}})
```

File: content/operations-guide/configuration/basic-configuration/_index.md [hunk 2]
```diff
@@ -15,6 +15,4 @@ This is the configuring the basic functionality in Humio.
 * [Configuring root access]({{< ref "root-access" >}})
 * [Configuring audit logging]({{< ref "audit-logging" >}})
 * [Configuring user authentication]({{< ref "user-authentication" >}}) 
-* Kafka configuration
-  - [Basic configuration for Kafka]({{< ref "kafka_configuration/" >}}) 
-  - [Changing Kafka servers]({{< ref "kafka-switch" >}}) 
+* [Basic configuration for Kafka]({{< ref "kafka_configuration/" >}})
```
Comment (mwl) [inline content/operations-guide/configuration/basic-configuration/_index.md]: When using [`ref`](https://gohugo.io/functions/ref/) it should be pointed towards a filename.

File: content/operations-guide/installation/bare-metal.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Disk space depends on the amount of ingested data per day and the number of rete
 
 _Retention days x GB injected / compression factor = needed disk space for a single server_
 
-For more information on retention in Humio, see [configuring data retention]({{< ref "retention.md" >}}). For more information on compression in Humio, see [index sizing]({{< ref "../../appendix/instance-sizing/" >}}).
+For more information on retention in Humio, see [configuring data retention]({{< ref "retention.md" >}}). For more information on compression in Humio, see [index sizing]({{< ref "instance-sizing" >}}).
```
Comment (mwl) [inline content/operations-guide/installation/bare-metal.md]: When using [`ref`](https://gohugo.io/functions/ref/) it should be pointed towards a filename.

File: content/operations-guide/installation/docker.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ Separate mount points help isolate Kafka from the other services. Kafka is notor
 consuming large amounts of disk space, so it's important to protect the other services from
 running out of disk by using a separate volume in production deployments.
 Make sure all volumes are being appropriately monitored as well. If your installation does
-run out of disk space and gets into a bad state, you can find recovery instructions [here]({{< ref "../configuration/basic-configuration/kafka-switch.md" >}}).
+run out of disk space and gets into a bad state, you can find recovery instructions [here]({{< ref "../administration/kafka-switch.md" >}}).
```
Comment (mwl) [inline content/operations-guide/installation/docker.md]: Avoid relative paths with [`ref`](https://gohugo.io/functions/ref/) and replace with file names. I.e.
```
… [here 🤬]({{< ref "kafka-switch.md" >}}).
```

… And "here" links.

File: content/operations-guide/configuration/basic-configuration/_index.md (hunk 2 above)
Comment (hobbes-the-tiger) [reply content/operations-guide/configuration/basic-configuration/_index.md]: Does this mean I should add .md to all of these if they're not directories?

File: content/operations-guide/installation/docker.md (hunk 4 above)
Comment (hobbes-the-tiger) [reply content/operations-guide/installation/docker.md]: So I don't need to tell it where the file is with ref links, even if it's in a different directory?

The "here" thing is a style guide thing, and that's something to discuss with @LH-ExcellentAdventure  :)

File: content/operations-guide/configuration/basic-configuration/_index.md (hunk 2 above)
Comment (mwl) [reply content/operations-guide/configuration/basic-configuration/_index.md]: I'll have to check regarding directories, but basically yes.

Comment (mwl) [reply content/operations-guide/configuration/basic-configuration/_index.md]: When it comes to directories, it _looks_ like you have to write the complete path, i.e.
```
[Administration]({{< ref "/operations-guide/administration" >}})
```

see https://gohugo.io/content-management/cross-references/

**Pull request merged**